flink-end-to-end-tests/test-scripts/kafka-common.sh

```diff
@@ -36,7 +36,7 @@ function setup_kafka_dist {
   mkdir -p $TEST_DATA_DIR
   KAFKA_URL="https://archive.apache.org/dist/kafka/$KAFKA_VERSION/kafka_2.11-$KAFKA_VERSION.tgz"
   echo "Downloading Kafka from $KAFKA_URL"
-  curl "$KAFKA_URL" --retry 10 --retry-max-time 120 > $TEST_DATA_DIR/kafka.tgz
+  curl ${KAFKA_URL} --retry 10 --retry-delay 2 --retry-max-time 120 --output ${TEST_DATA_DIR}/kafka.tgz
```

Comment:
I think an exponential backoff strategy is usually something good. Hence, I would suggest to remove `--retry-delay`.